devimint/src/tests.rs

```diff
@@ -2088,25 +2092,26 @@ pub async fn recoverytool_test(dev_fed: DevFed) -> Result<()> {
 
     debug!(target: LOG_DEVIMINT, ?utxos_descriptors, "recoverytool descriptors using UTXOs method");
 
-    let descriptors_json = [serde_json::value::to_raw_value(&serde_json::Value::Array(
-        utxos_descriptors
-            .iter()
-            .map(|d| {
-                let object = json!({
-                    "desc": d,
-                    "timestamp": 0,
-                });
-                object
-            })
-            .collect::<Vec<_>>(),
-    ))?];
+    let descriptors_json = serde_json::value::to_raw_value(&serde_json::Value::Array(vec![
+        serde_json::Value::Array(
+            utxos_descriptors
+                .iter()
+                .map(|d| {
+                    json!({
+                        "desc": d,
+                        "timestamp": 0,
+                    })
+                })
+                .collect(),
+        ),
+    ]))?;
```

Comment:
Note for reviewer: The double-array here looks strange, but is intentional. With only a single array, `recoverytool_tests` fails due to a malformatted request. The change here is required due to [this](https://github.com/apoelstra/rust-jsonrpc/pull/108) breaking change that was introduced in v0.17 of the `jsonrpc` crate.